examples/real_world_scenario.rs

```diff
@@ -0,0 +1,60 @@
+extern crate thread_profiler;
+
+#[cfg(feature = "thread_profiler")]
+use thread_profiler::ProfileScope;
+
+use std::thread;
+use std::string::String;
+use std::vec::Vec;
+
+fn main() {
+	if !cfg!(feature = "thread_profiler") {
+		panic!("This example must be run with the 'thread_profiler' feature enabled");
+	}
+	let workers = 10;
+	let mut tasks = Vec::new();
+	for i in 0..workers {
+		tasks.push(thread::spawn(move || {
```

Comment:
please use 4 spaces instead of tabs here (the convention used in rust/servo/wr/etc)

Reply:
crate is now rustfmt'd